tests/unit/test_charm.py

```diff
@@ -24,8 +33,8 @@ def test_unconfigured_charm_reaches_blocked_status():
 
 def test_misconfigured_port_charm_reaches_blocked_status():
     """
-    arrange: set up a charm.
-    act: trigger a configuration change with an invalid port.
+    arrange: set up a charm with an invalid port.
+    act: none
```

Comment:
Why act: none?

Reply:
There's no action. Once the charm is set up, it reaches blocked status